Dockerfile-linux-arm64

```diff
@@ -0,0 +1,120 @@
+FROM ubuntu:jammy
+
+ARG zlib_version
+ARG zlib_hash
+
+ARG libevent_version
+ARG libevent_hash
+
+ARG openssl_version
+ARG openssl_hash
+
+ARG tor_version
+ARG tor_hash
+
+ARG jobs
+
+COPY gpg-keys/tor.gpg /tor.gpg
+COPY gpg-keys/libevent.gpg /libevent.gpg
+COPY gpg-keys/openssl.gpg /openssl.gpg
+COPY patch /patch
+
+RUN apt-get update -qq && apt-get install --no-install-recommends -qq \
+  binutils-aarch64-linux-gnu \
+  build-essential \
+  curl \
+  ca-certificates \
+  gcc-aarch64-linux-gnu \
+  gnupg \
+  libc6-dev-arm64-cross \
+  python-is-python3 \
+  ; # end of apt-get install
+
+# Download and verify all tarballs
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
This doesn't have to be duplicated before each RUN, it will apply from the place where it's used on, until overridden with another SHELL

Reply:
Also, I'd go with
```Dockerfile
SHELL ["/bin/bash", "-e", "-o", "pipefail", "-c"]
```